src/lib/codemirror-editor.ts

```diff
@@ -26,7 +26,14 @@ import {
 // https://github.com/lezer-parser/javascript/issues/3. This module sets a
 // `CodeMirror` global.
 import '../_codemirror/codemirror-bundle.js';
+
+// TODO(aomarks) Provide an API for loading these themes dynamically. We can
+// include a bunch of standard themes, but we don't want them to all be included
+// here if they aren't being used.
 import codemirrorStyles from '../_codemirror/codemirror-styles.js';
+import monokaiTheme from '../_codemirror/themes/monokai.css.js';
+import ambienceTheme from '../_codemirror/themes/ambiance.css.js';
+import ayuMirageTheme from '../_codemirror/themes/ayu-mirage.css.js';
```

Comment:
Yeah, I think we should probably include one good light theme and one good dark theme. Anything else I was thinking we should add a `static CodeMirrorEditor.registerTheme(theme: CSSResult)` or similar API, so that the user can load themes however they want.

Though if we go the CSS property route, then loading themes would shift entirely to user-space.

I was going to punt for now, and do one of these two things in a followup, if that's ok.

Reply:
Uncompressed sizes:

default: 6.1K
monokai: 2.0K
ayu-mirage: 2.1K
ambiance: 26K

OK actually for now I'll remove ayu-mirage and ambiance. Ambiance is so big because it has an inline image.